Tuto-GUDHI-cover-complex.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>I have the following error on this line:</p><pre class="ql-syntax" spellcheck="false">Need coordinates for running differential coordinates!
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Input In [39], in &lt;cell line: 1&gt;()
----&gt; 1 compute_differential_coordinates(cover_complex, nodes=bnd[1])

Input In [32], in compute_differential_coordinates(M, nodes, features, sparse)
    213 if M.input_type == "distance matrix":
    214     print("Need coordinates for running differential coordinates!")
--&gt; 215     raise
    217 node_info = M.node_info_
    218 X = M.data

RuntimeError: No active exception to reraise

</pre><p>cover_complex.input_type is a distance matrix</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
thanks! fixed